tutorials/git.adoc

```diff
@@ -0,0 +1,313 @@
+total 12
+-rw-rw-r--. 1 syky syky 196 Sep 28 13:05 Jenkinsfile
+-rw-rw-r--. 1 syky syky 152 Sep 28 13:05 README.md
+-rw-rw-r--. 1 syky syky  25 Sep 28 13:10 USERNAME
+
+Git mi umožňuje podívat se na změny, které proběhly od minulého commitu:
+
+[source,console]
+----
+[syky@fedora B181AS-Username-syky27]$ git status
+On branch master
+Your branch is up to date with 'origin/master'.
+
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+
+	USERNAME
+
+nothing added to commit but untracked files present (use "git add" to track)
+----
+
+Vidím, že se vytvořil nový soubor `USERNAME`, řeknu Gitu,
+že chci, aby ho trackoval:
+
+
+[source,console]
+----
+[syky@fedora B181AS-Username-syky27]$ git add USERNAME
+[syky@fedora B181AS-Username-syky27]$ git status
+On branch master
+Your branch is up to date with 'origin/master'.
+
+Changes to be committed:
+  (use "git reset HEAD <file>..." to unstage)
+
+	new file:   USERNAME
+----
+
+Popíši změnu pomocí commitu:
+
+[source,console]
+----
+[syky@fedora B181AS-Username-syky27]$ git commit -am"USERNAME file with my username"
+
+*** Please tell me who you are.
+
+Run
+
+  git config --global user.email "you@example.com"
+  git config --global user.name "Your Name"
+
+to set your account's default identity.
+Omit --global to set the identity only in this repository.
+
+fatal: unable to auto-detect email address (got 'syky@fedora.(none)')
+----
+
+Pokud vidím tuto chybu, znamená to, že musím Gitu říci, kdo jsem. Čili:
+
+[source,console]
+$ git config --global user.email "sykorto6@fit.cvut.cz"
+$ git config --global user.name "Tomáš Sýkora"
+
+Poté zopakuji vytvoření commitu:
+
+[source,console]
+[syky@fedora B181AS-Username-syky27]$ git commit -am"USERNAME file with my username"
+[master 2078a00] USERNAME file with my username
+ 1 file changed, 1 insertion(+)
+ create mode 100644 USERNAME
+
+Commit je vytvořen, mohu se podívat, co se v něm změnilo:
+
+[source,console]
+----
+[syky@fedora B181AS-Username-syky27]$ git show
+commit 2078a007fa2d91e7bc2bb40d802bfa4eb17fd05c (HEAD -> master)
+Author: Tomáš Sýkora <sykorto6@fit.cvut.cz>
+Date:   Fri Sep 28 13:21:39 2018 +0200
+
+    USERNAME file with my username
+
+diff --git a/USERNAME b/USERNAME
+new file mode 100644
+index 0000000..ac11935
+--- /dev/null
++++ b/USERNAME
+@@ -0,0 +1 @@
++My super secret username
+----
+
+
+Následně nahraji změny na GitHub:
+
+[source,console]
+[syky@fedora B181AS-Username-syky27]$ git push
+Warning: Permanently added the RSA host key for IP address '192.30.253.113' to the list of known hosts.
+Counting objects: 3, done.
+Compressing objects: 100% (2/2), done.
+Writing objects: 100% (3/3), 362 bytes | 362.00 KiB/s, done.
+Total 3 (delta 0), reused 0 (delta 0)
+To github.com:3DprintFIT/B181AS-Username-syky27.git
+   dc5189b..2078a00  master -> master
+
+.Změny jsou na GitHubu vidět
+image::pushed_changes.png[]
+
+U repozitáře v Issues vidíte výsledek našeho systému,
```

Comment:
v záložce Issues - možná i nějak víc zvýraznit, že to hodnocení se nachází tady.